packages/7zip-15-05.vm/tools/chocolateyinstall.ps1

```diff
@@ -24,6 +25,10 @@ try {
   $7zExecutablePath = Join-Path $toolDir "$toolName.exe" -Resolve
   Install-BinFile -Name $toolName -Path $7zExecutablePath
 
+  # Make shortcut point to 7z File Manager so that it's more useful of a shortcut.
```

Comment:
I suggest to explain why it is more useful
```suggestion
  # Make shortcut point to 7z File Manager that has a UI instead of the command line 7z
```